<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonDialog</class>
 <widget class="QDialog" name="CommonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>481</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="P2_btn_ok">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>200</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>确    认</string>
     </property>
    </widget>
    <widget class="QLabel" name="P2_label_msg">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>481</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="P3_btn_cancel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>取    消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="P3_btn_ok">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>190</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>确    认</string>
     </property>
    </widget>
    <widget class="QLabel" name="P3_label_msg">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>481</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
